lib/beaker-pe/install/pe_utils.rb

```diff
@@ -899,7 +899,7 @@ def do_higgs_install host, opts
 
           #wait for output to host['higgs_file']
           #we're all done when we find this line in the PE installation log
-          higgs_re = /Please\s+go\s+to\s+https:\/\/.*\s+in\s+your\s+browser\s+to\s+continue\s+installation/m
+          higgs_re = /o\s+to\s+https:\/\/.*\s+in\s+your\s+browser\s+to\s+continue\s+installation/m
```

Comment:
Great point, yes, it will break the older versions because we just dropped the word "Please" in Davis build. I will update the PR and repush it quickly